CONTRIBUTING.md

```diff
@@ -1,79 +1,133 @@
 # Contributing to CodeBooker development
 
-We welcome Pull Requests (PR's) as well as having people working on fixing it's current issues. If you are coming to this project new.
-If you notice an issue with this app, please feel free to open an issue.
+This project welcomes contributors to help improve its codebase by identifying shortcomings in the project, highlighting bugs, and issues in the documentation.
 
-A lot of developers prefer asking if they could contribute or be assigned a task as a reply to a reported issue in the Issues tab. This is great.
+## What to contribute
 
-Typically, if you think it will take you longer than 72 hours after having been assigned to the issue you will work on to submit a PR, please
-let us know that way issues do not go stale.
+Anyone with basic programming skills down to grammer skills can contribute immensely to this project.
+Areas requiring contribution includes:
 
-Some tasks may not be beginner friendly even if tagged `good first issue`.. so try and judge the task accordingly.
-If you are a beginner, there are smaller tasks a beginner can work on such as style issues.
+- **Error checks**: Identify any mispellings or errors in documentation? 
 
-You should already be familiar with React 17 and or 18, JavaScript ES5/ES6/ES7.
+- **Documentation**: Possess good documentation skills: Good spelling, punctuation, knowledge of Markdown and grammar checker tool, and you believe you can help elevate this projects' documentation even better? 
 
-## Style
+- **Code contribution**: Identify a bug in this app or thought of a new feature that could plunge this app to a new dimension? 
+
+## How to contribute
+
+Developers often derive more convenience in seeking permission from the project to contribute, so these next headings will get you started in no time contributing.
```

Comment:
I can't able to understand on 1st read

Reply:
Yeah, that wording is confusing, and could probably be removed.

Reply:
> I can't able to understand on 1st read

Okay will work on this

Reply:
> Yeah, that wording is confusing, and could probably be removed.

Yeah i have simplified this, it reads better now.

Reply:
I just saw. Looks much better.  The original text was placed there because I had so many people in this (and other projects too) ask me if they could please work on a issue.  I understand that is just being polite and that was fine, but it is not necessary that people ask for permission to work on an issue.  I largely prefer that people just say they would like to work on an issue, and one of our maintaners can assign.

Reply:
Yes, i totally agree with that